Simplified/Simplified-Info.plist

```diff
@@ -23,7 +23,7 @@
 	<key>CFBundleSpokenName</key>
 	<string>Simply E</string>
 	<key>CFBundleVersion</key>
-	<string>1087</string>
+	<string>1385</string>
```

Comment:
We should try not to commit build number changes. If everyone does so, we'll always have a merge conflict when multiple developers are merging into the same branch.